composer.json

```diff
@@ -67,7 +68,12 @@
             "vendor/bin/ecs check bin packages src tests --fix",
             "bin/clean_levels.sh"
         ],
-        "phpstan": "vendor/bin/phpstan.phar analyse packages src tests --level max --configuration phpstan.neon"
+        "phpstan": "vendor/bin/phpstan.phar analyse packages src tests --level max --configuration phpstan.neon",
+        "build-phar": [
+            "vendor/bin/php-scoper add-prefix --output-dir=build --force",
```

Comment:
I would remove the dev dependencies first there

Reply:
I did that before but:

- it took so much time to remove, install dev, then install all
- it's covered by finder already
- it's not possible due to php-cs-fixer, that has `/src` file in `/tests` dir; which fails on `composer dump`

Reply:
The Finder doesn't cover that. You are ignoring a library test files with the current config but not the whole dev library.

Yeah PHP-CS-Fixer is a bit annoying here cf. https://github.com/api-platform/schema-generator/blob/master/scoper.inc.php#L39-L42 and https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3462

Although you are right you can also ignore this. This is something that will be handled in Box at some point which will be way more performant for PHP-Scoper as well as it will only scope the files filtered by Box